### DISCUSSION TITLE
error found in while running docker compose run

### DISCUSSION THREAD

**Discussion by GovindApollo** [apache/superset#28116] [General]
docker compose up -d
WARN[0000] The "CYPRESS_CONFIG" variable is not set. Defaulting to a blank string.
WARN[0000] The "CYPRESS_CONFIG" variable is not set. Defaulting to a blank string.
WARN[0000] The "SCARF_ANALYTICS" variable is not set. Defaulting to a blank string.
WARN[0000] /app/superset/docker-compose.yml: `version` is obsolete

**Comment by josedurigon** [comment]
I've had the same error running "docker compose pull".
